.github/workflows/build-and-publish-rule-book.yaml

```
@@ -123,7 +123,7 @@ jobs:
           extra_system_packages: "inkscape font-liberation font-hack"
           latexmk_use_lualatex: ${{ matrix.luaengine }}
           latexmk_shell_escape: true
-          post_compile: echo "icu_locale \"${{ matrix.language }}\"" >> index_style.ist && upmendex -s index_style main_${{ matrix.language }} && sed -i 's@\(\\noindent\\textbf{\)\(.\)@\\noindent\\textbf{\2}@g' main_${{ matrix.language }}.ind && lualatex --shell-escape main_${{ matrix.language }}.tex
+          post_compile: echo "icu_locale \"${{ matrix.language }}\"" >> index_style.ist && upmendex -s index_style main_${{ matrix.language }} && sed -i 's@\(\\noindent\\textbf{\)\(.\)@\\noindent\\textbf{\2}@g' main_${{ matrix.language }}.ind && ${{ matrix.luaengine && 'lualatex' || 'pdflatex' }} --shell-escape main_${{ matrix.language }}.tex
```

No, this won't do. GitHub doesn't support boolean type, both of those are true.
```suggestion
          post_compile: echo "icu_locale \"${{ matrix.language }}\"" >> index_style.ist && upmendex -s index_style main_${{ matrix.language }} && sed -i 's@\(\\noindent\\textbf{\)\(.\)@\\noindent\\textbf{\2}@g' main_${{ matrix.language }}.ind && ${{ matrix.luaengine == 'true' && 'lualatex' || 'pdflatex' }} --shell-escape main_${{ matrix.language }}.tex
```

---

https://docs.github.com/en/actions/learn-github-actions/expressions#literals and my test commit disagree with you